models.go

```diff
@@ -99,14 +99,14 @@ type NostrEvent struct {
 
 type Payment struct {
 	ID             uint `gorm:"primaryKey"`
-	AppId          uint `gorm:"index" validate:"required"`
+	AppId          uint `gorm:"index;index:idx_payment_sum" validate:"required"`
 	App            App  `gorm:"constraint:OnDelete:CASCADE"`
 	NostrEventId   uint `gorm:"index" validate:"required"`
 	NostrEvent     NostrEvent
-	Amount         uint
+	Amount         uint `gorm:"index:idx_payment_sum"`
```

Comment:
I think the amount in the index does not make sense to have the amount part of the index we do not query for that.

we do `WHERE app_id = ? AND preimage IS NOT ? AND created_at > ?`

so maybe we do the 3 in the index and include the amount that then can be used to sum things up.
https://atlasgo.io/guides/postgres/included-columns